src/main/java/pl/javastart/foodieapp/item/ItemService.java

```diff
@@ -0,0 +1,33 @@
+package pl.javastart.foodieapp.item;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@AllArgsConstructor
```

Comment:
zeby wstrzyknelo beany to musi byc @RequiredArgsConstructor

Reply:
zrobione